### PR TITLE
[DatePicker] Added the possibilitie to pass custom prefix and suffix icons

### PR DESCRIPTION
**Pull request by AdamSGit**
Added the possibilitie to pass custom prefix and suffix icon class with same props as usual (prefix-icon and suffix-icon)

Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md) | [Español](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.es.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

**Comment by Leopoldthecoder** [comment]
Hi, thanks for contributing!

The prefix icon looks good to me, but there seem to be some unexpected behavior regarding suffix icon:
untitled

This is unintentional, right?

**Comment by AdamSGit** [comment]
@Leopoldthecoder No, this is intentional, the clear icon should be customized as well. But I guess what disturb you is the name "suffix-icon" instead of maybe "clear-icon" ? I did this to better match element's conventions. Let me know if you want me to change it.

**Comment by Leopoldthecoder** [comment]
Oh, sorry for the misunderstanding. I think `clear-icon` is better since this prop is used to replace the default clear icon. So could you change it?

**Comment by AdamSGit** [comment]
@Leopoldthecoder Done. I also better updated the 3 pickers pages on english doc.

File: packages/date-picker/src/picker.vue [hunk 1]
```diff
@@ -53,6 +53,7 @@
       @input="handleStartInput"
       @change="handleStartChange"
       @focus="handleFocus"
+      :prefix-icon="triggerClass"
```
Comment (Leopoldthecoder) [inline packages/date-picker/src/picker.vue]: This line and line 69 seem to be unnecessary.

**Comment by AdamSGit** [comment]
@Leopoldthecoder You are right those lines weren't usefull, I cleaned them.

**Comment by whh2017** [comment]
good！

**Comment by Jean-Jacques-Nuoan** [comment]
what should i do when i want  to move the prefix-icon to the end,instead of the suffix-icon?

**Pull request merged**